modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1867,12 +1880,22 @@ private void resizeCellSize(T cell) {
         }
     }
 
-    private List<T> getCells() {
+    /**
+     * @return The list of cells representing those cells which actually make up the
+     * * current view. The cells are ordered such that the first cell in this
+     * * list is the first in the view, and the last cell is the last in the
+     * * view. When pixel scrolling, the list is simply shifted and items drop
+     * * off the beginning or the end, depending on the order of scrolling.
+     */
+    protected List<T> getCells() {
```

Comment:
The primary intent of overriding the VirtualFlow is to gain access to the rows displayed and tweak them. Therefore, this getter should be protected. 
For example, I use it to sort the rows that allow me to strictly control how the rows will be displayed, and which row will overlap another row when some cell are spanning.